common/app/navigation/helpers/helpers.scala

```diff
@@ -10,8 +11,8 @@ object NavigationHelpers {
     val editionId = edition.id.toLowerCase()
```

Comment:
I think this should be called `getReaderRevenueLinks` now

It could also be worth double-checking whether it's still needed. For instance, `editionId` is never used so it looks like whatever country-specific logic it once had is gone.

My thought would be: either make this so it can handle all the places where we're putting reader revenue links (header AND footer), or get rid of it and just manually put in the appropriate links wherever this is called (which is just two places). @NataliaLKB could be a great person to ask since she knows a lot about the header and about AMP (which is the second use-case of this function)

Reply:
> manually put in the appropriate links wherever this is called (which is just two places

Or create a variable which can be used in those two places just to make sure AMP pages are updated at the same time as regular articles. They are often missed